roles/openshift_master/tasks/system_container.yml

```diff
@@ -26,11 +27,51 @@
   when:
     - ("master" in result.stdout)
     - (openshift.common.version is not defined) or (openshift.common.version != openshift_version) | bool
+    - openshift.master.ha is not defined or not openshift.master.ha | bool
 
 - name: Install Master system container package
   command: >
     atomic install --system --name={{ openshift.common.service_type }}-master {{ openshift.common.system_images_registry }}/{{ openshift.master.master_system_image }}:{{ openshift_image_tag }}
   when:
     - (openshift.common.version is not defined) or (openshift.common.version != openshift_version) or ("master" not in result.stdout) | bool
+    - openshift.master.ha is not defined or not openshift.master.ha | bool
+  notify:
+    - restart master
+
+# HA
+- name: Update Master HA system container package
+  command: >
+    atomic containers update {{ openshift.common.service_type }}-master-{{ item }}
+  register: update_result
+  changed_when: "'Extracting' in update_result.stdout"
+  with_items:
+    - api
+    - controllers
+  when:
+    - ("master" in result.stdout)
```

Comment:
Not an issue for this PR but ``result`` is a little confusing as it doesn't describe what is actually being checked. No change required here but it would be nice in another PR to rename ``result``.